tests/e2e/steps_permissionless_ics.go

```diff
@@ -0,0 +1,213 @@
+package main
+
+import (
+	"time"
+
+	clienttypes "github.com/cosmos/ibc-go/v8/modules/core/02-client/types"
+	e2e "github.com/cosmos/interchain-security/v6/tests/e2e/testlib"
+)
+
+// stepsPermissionlessICS tests
+// - starting multiple permissionless consumer chains with the same chain ID
+// - opt-in of a validator on two different chains with the same chain ID
```

Comment:
Mention that it tests that this "does not work" since it is not allowed. Someone reading it might think that this is possible.